.ci/azure-pipelines.yml

```diff
@@ -10,38 +24,42 @@ jobs:
         persistCredentials: true
 
       - task: CmdLine@2
-        displayName: "Clone Server"
+        displayName: "Clone Server Branch"
+        condition: in(variables['Build.Reason'], 'IndividualCI')
+        inputs:
+          script: "git clone --single-branch --branch $(Build.SourceBranchName) --depth=1 https://github.com/jellyfin/jellyfin.git $(Build.SourcesDirectory)/jellyfin"
+
+      - task: CmdLine@2
+        displayName: "Clone Server Target"
+        condition: in(variables['Build.Reason'], 'PullRequest')
         inputs:
-          script: "git clone --single-branch --branch master --depth=1 https://github.com/jellyfin/jellyfin.git $(Build.SourcesDirectory)/jellyfin"
+          script: "git clone --single-branch --branch $(System.PullRequest.TargetBranch) --depth=1 https://github.com/jellyfin/jellyfin.git $(Build.SourcesDirectory)/jellyfin"
 
       - task: CmdLine@2
-        displayName: "Clone Web Client (Master, Release, or Tag)"
-        condition: and(succeeded(), or(contains(variables['Build.SourceBranch'], 'release'), contains(variables['Build.SourceBranch'], 'master'), contains(variables['Build.SourceBranch'], 'tag')), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'BuildCompletion'))
+        displayName: "Clone Web Branch"
+        condition: in(variables['Build.Reason'], 'IndividualCI')
```

Comment:
Why no `BatchedCI`? That can happen if someone submits multiple commits to a PR at the same time and only the last one is built. And if this one is triggered by another that is what `BuildCompletion` is, and we might trigger this build when the server is done.

Reply:
I'm not sold on the benefits of batched CI for such an inactive repository.

Reply:
It's the default though, so better to just use the exact same stuff everywhere so we don't get slightly different behaviors in similar steps.